README.md

```diff
@@ -99,16 +101,20 @@ The next step is to build an application you can combine with your bootloader to
     mbed compile -m <TARGET_NAME> -t ARM
     ```
 
+It creates two binary images in the `BUILD/<TARGET_NAME>/<TOOLCHAIN_NAME>` directory. The image that contains the combined bootloader and application (suitable for flashing to the device's ROM directly) is saved with the name `<project-name.bin>`. The image that should be used as the update binary (the one that should be placed on the SD card) is saved with the name `<project-name>_application.bin`.
```

Comment:
Specifically, please change the third sentence to "Use the image saved with the name `<project-name>_application.bin' as the update binary (the one you place on the SD card)."